Sources/OAApplicationMode.m

```diff
@@ -540,7 +554,7 @@ - (OAApplicationModeBean *) toModeBean
 + (void) onApplicationStart
 {
     [self initCustomModes];
-//    [self initModesParams];
+    [self initModesParams];
```

Comment:
This is a wrong place to do it. Set it on the line from the initial comment. (145)